cirq/ops/common_channels.py

```diff
@@ -36,8 +36,8 @@ def __init__(self,
                  p_x: Optional[float] = None,
                  p_y: Optional[float] = None,
                  p_z: Optional[float] = None,
-                 error_probabilities: Optional[Dict[str, float]] = None
-                ) -> None:
+                 error_probabilities: Optional[Dict[str, float]] = None,
+                 tol: Optional[float] = None) -> None:
```

Comment:
Since below you make `None` equivalent to 1e-6, we don't actually need this to be an optional type. Let's change type to `float` and write the default explicitly.